packages/react-router/tests/redirects.test.tsx

```diff
@@ -463,88 +459,3 @@ describe('router.navigate navigation using layout routes resolves correctly', as
     expect(router.state.location.pathname).toBe('/g/tkdodo')
   })
 })
-
-function createContextRouter<TCtx extends Record<string, unknown>>(
```

Comment:
Removed since this is no longer representative of what's actually happening in the browser.